pathod/language/base.py

```diff
@@ -105,7 +108,7 @@ def __repr__(self):
 class _TokValueLiteral(Token):
 
     def __init__(self, val):
-        self.val = val.decode("string_escape")
```

Comment:
We use string escape here so that users can supply escaped strings which are then treated as bytes - I'm afraid we can't just leave it out.

(I believe we can do that in a later PR though, this is not pathod.utils)